features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,118 @@
+# Features module: how to share site’s config with others?
+
+If you already had a chance to build one or two Drupal sites,  you probably know Drupal site consists of three parts:
```

Comment:
Neither 😜 

I think that ones with the blue circles that Orit & you created are better, because:

* First ones looks boring, and misses the human touch
* For consistency, I think we should provide a similar look and feel.